actions/contacts/search.go

```diff
@@ -91,26 +92,51 @@ func (a *Action) search(c *gin.Context) {
 func (a *Action) getContacts(c *gin.Context) {
```

Comment:
Please update your branch and resolve conflicts - because, currently there is no such concept as `(a *Action)` for handlers.

Reply:
done